.typescript-declarations/St.d.ts

```diff
@@ -1,9 +1,8 @@
 declare namespace imports.gi.St {
-	export class Widget extends Clutter.Actor {
-		constructor(options?: any);
+
+	interface WidgetWithoutParent {
```

Comment:
Just call these `I{classname}` instead `{classname}WithoutParent`. They are declaring the properties going into it's class.